### ISSUE TITLE
[BUGFIX] Volarkite with Real First Person 2 rendering head incorrectly 

### ISSUE THREAD

**Issue by ghost** [TheCyberBrick/The-Betweenlands#1124]
Thought i'd give it a try and request a fix here since the author of Real First Person 2 seems MIA at the moment-
Is it possible to get a compatibility fix for the mod, or at least a config to solve this? thanks in advance!

#### Betweenlands version:
TheBetweenlands-3.7.1

#### Forge version:
MC 1.12.2
forge-14.23.5.2854

#### Singleplayer or Multiplayer:
Singleplayer

#### Installed mods:
TheBetweenlands
Real First Person 2

#### How to reproduce:

1. Have Real First Person 2 Installed
2. Use Volarkite
3. Look around in first person 

#### Expected result:
Basically similar/same behavior,  just without rendering the head.

#### Actual result:
The head renders in first person, despite Real First Person 2 supposedly stopping the render of the head in first person.

#### Pictures:
![image](https://user-images.githubusercontent.com/52370288/98628041-99e87100-22e3-11eb-8dac-dae07c958df7.png)
